platform/darwin/src/MGLHillshadeStyleLayer.h

```diff
@@ -24,8 +24,15 @@ typedef NS_ENUM(NSUInteger, MGLHillshadeIlluminationAnchor) {
 };
 
 /**
- Client-side hillshading visualization based on DEM data. Currently, the
- implementation only supports Mapbox Terrain RGB tiles
+ An `MGLHillshadeStyleLayer` is a style layer that renders raster <a
+ href="https://en.wikipedia.org/wiki/Digital_elevation_model">digital elevation
+ model</a> (DEM) tiles on the map.
```

Comment:
From that Wikipedia article, I gathered that _digital elevation model_ is a generic term, encompassing other formats like USGS DEM. Would Terrarium be considered a DEM format?

Reply:
We should avoid being overly broad here, especially if it’s suggesting something we don’t currently support.

Reply:
It’s true that we should avoid overpromising, but I think it’s also important to draw a connection to the MGLRasterDEMSource class, which does use the same terminology, since the type system doesn’t otherwise guide the developer towards using MGLHillshadeStyleLayer with MGLRasterDEMSource.